### ISSUE TITLE
beforeCommand, afterCommand do not work when use async

### ISSUE THREAD

**Issue by elanciel** [webdriverio/webdriverio#4849]
Hello,

I'm trying to use beforeCommand and afterCommand hooks in order to catch browser.url command. unfortunately, it does not work.

```
    /**
     * Runs before a WebdriverIO command gets executed.
     * @param {String} commandName hook command name
     * @param {Array} args arguments that command would receive
     */
    beforeCommand: function (commandName, args) {
//something
    },

    /**
     * Runs after a WebdriverIO command gets executed
     * @param {String} commandName hook command name
     * @param {Array} args arguments that command would receive
     * @param {Number} result 0 - command success, 1 - command error
     * @param {Object} error error object if any
     */
     afterCommand: function (commandName, args, result, error) {
//something
     },

```

Did I miss something ?

Thx

**Comment by mgrybyk** [comment]
It doesn't work in async mode

**Comment by christian-bromann** [comment]
While this is not a regression I think we should make before/after command work with our without having `@wdio/sync` installed. The experience has to be the same.

**Comment by elanciel** [comment]
Thanks @mgrybyk and @christian-bromann for your feedback.
I will use @wdio/sync waiting for a fix :)

**Comment by christian-bromann** [comment]
fixed in #4853

**Comment by elanciel** [comment]
I have tested the v5.16.12 but unfortunately i have these tons of errors :(
I have removed @wdio/sync and upgrade version 5.16.11 to 5.16.12


```
[0-0] (node:59722) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'options' of undefined
[0-0] (node:59722) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
[0-0] (node:59722) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
[0-0] (node:59722) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'options' of undefined
```

**Comment by christian-bromann** [comment]
@elanciel you have to provide a reproducible example, these errors can be cause by a lot of things.

**Comment by elanciel** [comment]
Node: v10.16.3
I can reproduce it on this simple code without using any hooks :

```
'use strict';
const {expect} = require('chai');
describe('Without ui5', function () {
    it('Dummy test to validate iFrame', async() => {
        await browser.url('https://developer.mozilla.org/fr/docs/Web/HTML/Element/iframe');
        const frame = await $('.tabbed-standard');
        await browser.switchToFrame(frame);
        const btnHTLM = await $('#html');
        await btnHTLM.waitForDisplayed({ timeout: 3000 });
        expect(await btnHTLM.getText()).to.equal('HTML');
        await browser.switchToParentFrame();
        const pageTitle = await $('h1');
        expect(await pageTitle.getText()).to.equal('<iframe>');
    });
});

```

**Comment by erwinheitzman** [comment]
@christian-bromann I was able to reproduce this as well. The following setup can be used for reproduction: https://github.com/erwinheitzman/jest-webdriverio-standalone-boilerplate

**Comment by christian-bromann** [comment]
Let's reopen then even though it seems to be a different issue.

**Comment by mgrybyk** [comment]
@christian-bromann this is a regression since `5.16.11` and now standalone mode doesn't work at all.

**Comment by christian-bromann** [comment]
working on a fix

**Comment by christian-bromann** [comment]
Can someone review #4858?

**Comment by christian-bromann** [comment]
Fixed with version `v5.16.13`

**Comment by elanciel** [comment]
it works perfectly ! thanks @christian-bromann
